dev/integration_tests/android_host_app_v2_embedding/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-REPLACEME-all.zip
```

Comment:
was REPLACEME a note to self to change? or are we interpolating this somehow?

Reply:
It is the string we look for when trying to find what to replace.

Reply:
SGTM